src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p className="Person__age">
+      {person.age && `I am ${person.age}`}
+    </p>
+    <p className="Person__partner">
```

Comment:
If the person is not married, print the message 'I am not married'. Use a conditional expression to handle this case.